setup.cfg

```diff
@@ -80,7 +80,7 @@ all =
     pytz
     jplephem
     mpmath
-    asdf>=2.6
+    asdf>=2.9.2
```

Comment:
Is this too controversial for backport, @nden , @WilliamJamieson , or @eslavich ?

Reply:
I don't think so, the worst thing someone would encounter upgrading from 2.6 --> 2.9.2 is a deprecation warning.  We've been pretty good about staying true to semantic versioning.

Reply:
Note: Looks like it is only tagged, not released yet.